app/assets/stylesheets/shared/form.scss

```diff
@@ -347,7 +347,7 @@ small {
   }
 }
 
-.prx-help {
+.prx-input-group-text {
```

Comment:
I renamed this selector (and replaced across the app) since its used for more than the help button.